nilearn/plotting/displays.py

```diff
@@ -1242,37 +1242,45 @@ def find_cut_coords(self, img=None, threshold=None, cut_coords=None):
                           for c in sorted(self._cut_displayed)]
         return cut_coords
 
+    def _find_inital_axes_coord(self,index):
+        "find coordinates for inital axes placement for xyz cuts"
+
+        rect_x0, rect_y0, rect_x1, rect_y1 = self.rect
+
+        if (index == 0):
+                coord1 = rect_x1 - rect_x0
+                coord2 = 0.5 * (rect_y1-rect_y0) + rect_y0
+                coord3 = 0.5 * (rect_x1 - rect_x0) + rect_x0
+                coord4 = rect_y1-rect_y0
+
+        if (index == 1):
+                coord1 = 0.5 * (rect_x1 - rect_x0) + rect_x0
+                coord2 = 0.5 * (rect_y1-rect_y0) + rect_y0
+                coord3 = rect_x1 -rect_x0
+                coord4 = rect_y1-rect_y0
+        if (index == 2):
+                coord1 = rect_x1 - rect_x0
+                coord2 = rect_y1 -rect_y0
+                coord3 = 0.5 * (rect_x1 - rect_x0) + rect_x0
+                coord4 = 0.5 * (rect_y1-rect_y0) + rect_y0
+ 
+        return([coord1,coord2,coord3,coord4])
+
     def _init_axes(self, **kwargs):
+
         cut_coords = self.cut_coords
         if len(cut_coords) != len(self._cut_displayed):
             raise ValueError('The number cut_coords passed does not'
                              ' match the display_mode')
-        x0, y0, x1, y1 = self.rect
+
         facecolor = 'k' if self._black_bg else 'w'
+        
         # Create our axes:
```

Comment:
This comment is unnecessary: the function name & code are telling what's ahppenning.